README.md

```diff
@@ -88,7 +88,6 @@ incomplete (lacks the `type` setting) and `persistent_state` is `present`,
 the behavior is undefined. The value `absent` makes the role ensure
 that the profile is not present on the target host.
 
-
```

Comment:
Sorry that was a mistake. The commit should be dropped.

Reply:
ok, removed it